src/java/org/apache/cassandra/service/accord/AccordCommandsForKey.java

```diff
@@ -364,13 +374,13 @@ public void updateSummaries(AccordCommand command)
             if (command.status.previous() == null || !command.status.previous().hasBeen(Status.Committed))
                 uncommitted.map.blindRemove(command.txnId());
 
-            ByteBuffer bb = AccordPartialCommand.WithDeps.serializer.serialize(command);
-            committedById.map.blindPut(command.txnId(), bb);
-            committedByExecuteAt.map.blindPut(command.executeAt(), bb);
+            ByteBuffer partialCommand = AccordPartialCommand.serializer.serialize(key, command);
+            committedById.map.blindPut(command.txnId(), partialCommand);
```

Comment:
Probably, yes. I thought I may have even had a TODO somewhere about this (or perhaps some other memory accounting).

I'm not sure it is worth agonising over this too much since 1) this is going to be pessimistic; 2) it's hard to cleanly avoid; 3) this is likely to be modified further before long, in a manner that may obviate this